addons/account/models/account_analytic_account.py

```diff
@@ -25,7 +25,7 @@ def _compute_invoice_count(self):
         ])
         query.add_where(
             'account_move_line.analytic_distribution ?| array[%s]',
-            [str(account_id) for account_id in self.ids],
+            [[str(account_id) for account_id in self.ids]],
```

Comment:
This will generate stuff like `account_move_line.analytic_distribution ?| array[ARRAY[1, 2, 3]]`.
You have to replace `array[%s]` by `%s` in the query string.

Reply:
Wow, apparently `ARRAY[ARRAY[...]]` is processed just like `ARRAY[...]`.
SQL is so crappy that this works by chance/accident :eyes:

Reply:
updated